ledger-core/rms/proto_vnvn.r0.proto

```diff
@@ -1,15 +1,28 @@
 syntax = "proto3";
 
-package payload;
+package rms;
 
 import "github.com/gogo/protobuf/gogoproto/gogo.proto";
 import "github.com/insolar/assured-ledger/ledger-core/insproto/ins.proto";
-import "insolar/payload/localtypes.proto";
+import "rms.proto";
 
 option (insproto.notation_all) = true;
 option (insproto.zeroable_all) = true;
 option (gogoproto.goproto_enum_prefix_all) = false;
 option (insproto.register_all) = "RegisterMessageType";
+option (gogoproto.goproto_getters_all) = true;
+option (gogoproto.equal_all) = true;
+
+message Meta {
+    option (insproto.id) = 1000;
+
+    Any Payload = 20;
+    Binary Sender = 21 [(gogoproto.customtype) = "github.com/insolar/assured-ledger/ledger-core/reference.Global", (insproto.zeroable) = false];
```

Comment:
Should be Reference and without customtype

Reply:
fixed in https://github.com/insolar/assured-ledger/pull/766